### PR TITLE
Doctor space man keebs

### PR DESCRIPTION
**Pull request by DJZenose**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        if (clockwise) {
+            tap_code_delay(KC_VOLU, 10);
+        } else {
+            tap_code_delay(KC_VOLD, 10);
+        }
+    }
+    return false;
```
Comment (drashna) [inline keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c:30]:
```suggestion
```

This should be moved to an "andromeda.c" file 

This allows for a default behavior to be defined, without the need for the user to specify the behavior.  This is very useful for the configurator which doesn't have a way to configure this behavior, currently.

```suggestion
#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if (clockwise) {
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
        }
    }
    return true;
}
#endif
```

File: keyboards/doctorspaceman/andromeda/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# doctorspaceman/andromeda
+
+![doctorspaceman/andromeda](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Connor McQuade](https://github.com/DJZenose)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (drashna) [inline keyboards/doctorspaceman/andromeda/readme.md:9]: These need to be filled out.

File: keyboards/doctorspaceman/skirmisher/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/doctorspaceman/skirmisher/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/doctorspaceman/skirmisher/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# skirmisher
+
+![skirmisher](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Jeremy Allan](https://github.com/Doctor-Space-Man)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (drashna) [inline keyboards/doctorspaceman/skirmisher/readme.md:9]: needs to be filled out.

File: layouts/default/ortho_5x5/default_ortho_5x5/keymap.c [hunk 6]
```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline layouts/default/ortho_5x5/default_ortho_5x5/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/doctorspaceman/andromeda/readme.md (hunk 3 above)
Comment (Doctor-Space-Man) [reply keyboards/doctorspaceman/andromeda/readme.md:9]: I'll get right on them

File: layouts/default/ortho_5x5/default_ortho_5x5/keymap.c (hunk 6 above)
Comment (Doctor-Space-Man) [reply layouts/default/ortho_5x5/default_ortho_5x5/keymap.c:1]: Can do

File: keyboards/doctorspaceman/skirmisher/readme.md (hunk 5 above)
Comment (Doctor-Space-Man) [reply keyboards/doctorspaceman/skirmisher/readme.md:9]: No problem!

File: keyboards/doctorspaceman/skirmisher/keymaps/default/keymap.c (hunk 4 above)
Comment (Doctor-Space-Man) [reply keyboards/doctorspaceman/skirmisher/keymaps/default/keymap.c:1]: Sure thing.

File: keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c (hunk 2 above)
Comment (Doctor-Space-Man) [reply keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c:30]: Thanks for the advice!

File: keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c (hunk 1 above)
Comment (Doctor-Space-Man) [reply keyboards/doctorspaceman/andromeda/keymaps/default/keymap.c:1]: Will do!

**Pull request closed without merging**